tests/HttpAdapterTest.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
-namespace Http\Adapter\Tests;
+namespace Http\Adapter\Guzzle6\Tests;
 
 use GuzzleHttp\Client;
-use Http\Adapter\Guzzle6HttpAdapter;
+use Http\Adapter\Guzzle6\Client as Adapter;
```

Comment:
I think this is something that should be discussed generally, as it affects the whole Adapter naming idea. Can you open an issue in httplug to discuss this?